core/native.gradle

```diff
@@ -95,6 +97,7 @@ model {
 
                 }
                 if (toolChain in VisualCpp) {
+                    linker.args "/LIBPATH:$NETFXKitsDir\\Lib\\um\\${targetPlatform.name}"
```

Comment:
No, it's not covered by winSdkDir. .NET FX is a separate set of libraries that is no longer included in Windows SDK since Visual Studio 2017. It used to be part of Windows 7.1A SDK, and we can't really tell we depend on Windows 8 SDK (which is an optional SDK), since .NET FX SDK is the SDK for C++/CLI and .NET Framework development - there's a separate variable for that in Visual Studio. That's why it'd make sense to have the same variable in the build script. It explicitly clarifies what .NET Framework version SDK we're linking against. This file exists also in 4.7 , but the build script assumes that as a minimum we require .NET Framework 4.6